### PR TITLE
FE completed

### PR DESCRIPTION
**Pull request by Kwyfarrah**
截圖 2021-10-24 18 55 23
截圖 2021-10-24 18 54 47
截圖 2021-10-24 18 54 10
截圖 2021-10-24 18 54 04

### CODE REVIEW

**Comment by Kwyfarrah** [comment]
Needs Bundle install

File: lib/templates/erb/scaffold/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<%# frozen_string_literal: true %>
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
+  <%%= f.error_notification %>
```
Comment (darren987469) [inline lib/templates/erb/scaffold/_form.html.erb:3]: `<%=` is enough?

File: app/views/games/new.html.erb [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+<div class="container mx-auto">
+  <img
+    src="https://images.unsplash.com/photo-1598100223749-1df46ef2ddac?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1335&q=80&auto=format&fit=crop&w=634&q=80"
+    alt="image">
+  <div class="container-text">
+    <h1>New Game</h1>
+    <p>Start here and sharp the conversation</p>
+    <%= render 'form', game: @game %>
+    <%= link_to 'All Game Lists', games_path %>
+    <span>© 2021 darren987469 All Rights Reserved</span>
```
Comment (darren987469) [inline app/views/games/new.html.erb:10]: extra partial view?

**Review by darren987469 — APPROVED** [review]
Greate job! Looks totally different!

**Pull request merged**